src/main.c

```diff
@@ -279,13 +279,30 @@ static gchar *info_formatter_shell(RaucManifest *manifest)
 	g_string_append_printf(text, "RAUC_MF_IMAGES=%d\n", g_list_length(manifest->images));
 	g_string_append_printf(text, "RAUC_MF_FILES=%d\n", g_list_length(manifest->files));
 
+	g_string_append_printf(text, "RAUC_MF_HOOKS=");
+	if (manifest->hooks.install_check == TRUE) {
+		g_string_append_printf(text, "install-check;");
```

Comment:
Semicolons are not easy to parse in shell and we already use space separated lists elsewhere, so we should just use spaces here.